packages/dataviews/src/filter-summary.js

```diff
@@ -167,19 +190,81 @@ export default function FilterSummary( {
 				toggleRef.current?.focus();
 			} }
 			renderToggle={ ( { isOpen, onToggle } ) => (
-				<Button
-					__experimentalIsFocusable
-					size="compact"
-					onClick={ onToggle }
-					aria-expanded={ isOpen }
-					ref={ toggleRef }
-				>
-					<FilterText
-						activeElement={ activeElement }
-						filterInView={ filterInView }
-						filter={ filter }
-					/>
-				</Button>
+				<div className="dataviews-filter-summary__chip-container">
+					<Tooltip
+						text={ sprintf(
+							/* translators: 1: Filter name. */
+							__( 'Filter by: %1$s' ),
+							filter.name.toLowerCase()
+						) }
+						placement="top"
+					>
+						<div
+							className={ classnames(
+								'dataviews-filter-summary__chip',
+								{
+									'has-reset': canResetOrRemove,
+									'has-values': hasValues,
+									'is-primary': isPrimary,
```

Comment:
What is this class for? I don't see it in the CSS anywhere. There's one other case.